src/handlers.js

```diff
@@ -247,6 +247,13 @@ async function jobHandler(message) {
   debug('Received message. Starting processing...');
   let context = this.context;
 
+  this.monitor=new Date();
```

Comment:
Really we would like to use `this.monitor` to report the calculation we do. So it would be more something like `var elapsed = Date.now() - message.payload.created` and then using [the measure method of monitor](https://github.com/taskcluster/taskcluster-lib-monitor#measuring-and-counting-things)  to report it! You may have to do a bit more to make it work. This is just an outline of what I think must be done :)